gluon-connect-file-provider/src/main/java/com/gluonhq/samples/connect/file/FileListView.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2016, 2018 Gluon
```

Comment:
2016, 2020,

(and review other files)

Reply:
Since this file hasn't been updated, I refrained from updating the header. I have updated the header [here](https://github.com/gluonhq/gluon-samples/pull/42/files?file-filters%5B%5D=.java&file-filters%5B%5D=.md&file-filters%5B%5D=.png#diff-3c034f47b70c2ed64e3e42f5ed29ec8fR2). I am not sure what is the ultimate goal, but if we keep the commits from earlier then it makes sense to keep the header as it is, as all we are doing to the project is swapping out Gradle files with Maven's pom.xml with necessary updates to a few files.